.github/workflows/main.yml

```diff
@@ -2,9 +2,19 @@ name: Node.js GitHub Action
 
 on:
   push:
-    branches:
-      - main
+    branches: [main]
+    path-ignore:
+      - '.gitignore'
+      - '*.md'
+      - '*.txt'
+
+      
   pull_request:
+    branches: [main]
+    path-ignore:
+      - '.gitignore'
+      - '*.md'

```

Comment:
Why to ignore .md files?

Reply:
why we will keep it? node ci is for our front-end okay so does't make it sense to run ci whenever someone change `.md` files. simple :)